lib/persistence/object.rb

```diff
@@ -150,51 +150,40 @@ def process_two_phase_pending_objects
     #                             :edit_sequence => '12312312321'}
     #                             :extra_data => { ... }, ]
     def update_objects_with_query_results(objects_to_be_renamed)
-      # puts "Objects to be renamed: #{objects_to_be_renamed}"
-
       puts({connection_id: config[:connection_id], method: "update_objects_with_query_results", objects_to_be_renamed: objects_to_be_renamed})
 
       prefix = "#{path.base_name}/#{path.ready}"
       prefix_with_bucket = "#{path.base_name_w_bucket}/#{path.ready}"
 
       puts({connection_id: config[:connection_id], method: "update_objects_with_query_results", prefix: prefix, prefix_with_bucket: prefix_with_bucket})
 
-
-       # files = amazon_s3.bucket.objects(prefix: prefix)
-       #
-       # puts "Files in bucket: #{files}"
-       # puts "Files in bucket: #{files.first}"
-       #
-       # unless files.first
-       #   puts " No Files to be updated at #{prefix}"
-       #   return
-       # end
-
       objects_to_be_renamed.to_a.compact.each do |object|
-        filename     = "#{prefix}/#{object[:object_type].pluralize}_#{sanitize_filename(object[:object_ref])}_"
-        filename_with_bucket = "#{prefix_with_bucket}/#{object[:object_type].pluralize}_#{sanitize_filename(object[:object_ref])}_"
-
+        s3_object = nil
+        filename     = "#{prefix}/#{object[:object_type].pluralize}_#{sanitize_filename(object[:list_id])}_"
```

Comment:
This line could also be written as

```
        filename = "%s/%s_%i_" % [
          prefix,
          object[:object_type].pluralize,
          sanitize_filename(object[:list_id])
        ]
```

Of course it's a matter of opinion, but I think it helps see the "dash-underscore-underscore" pattern.

Plus, the `%i` means "integer", so this line will break if `sanitize_filename(object[:list_id])` is not an integer.

Reply:
`sanitize_filename(object[:list_id])` could be an integer or it could not be an integer. I was trying to not assume what type the item was. Doesn't the `"#{whatever}"` automatically call `.to_s` on whatever?

Reply:
Yup, it does call `to_s`, as does `%s` on the other syntax. Don't want to derail this PR, just offering an alternate syntax -- but one person's "more readable"  is another person's "Egyptian doodles". :-)